### DISCUSSION TITLE
😄 v0.15.0

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#2720] [Versions & Releases]
## What's Changed
* Update stochastic_duration_predictor.py by @mengting7tw in https://github.com/coqui-ai/TTS/pull/2663
* Fix Tortoise load by @erogol in https://github.com/coqui-ai/TTS/pull/2697
* Inference API for 🐶Bark by @erogol in https://github.com/coqui-ai/TTS/pull/2685
* Drop Python 3.7 and 3.8 and stage Python 3.11 by @erogol in https://github.com/coqui-ai/TTS/pull/2700

**Full Changelog**: https://github.com/coqui-ai/TTS/compare/v0.14.3...v0.15.0

<hr /><em>This discussion was created from the release <a href='https://github.com/coqui-ai/TTS/releases/tag/v0.15.0'>😄 v0.15.0</a>.</em>

**Comment by AWAS666** [comment]
Didn't want to create a full issue for this, but the docs for bark are missing.
Readme links to it, but it just 404s.
Missing in repo aswell, no bark.md in docs/models

**Reply by erogol** [reply]
Fixed now 👍
